pallets/common/src/constants.rs

```diff
@@ -40,6 +40,16 @@ pub mod queue_priority {
 
     /// Queue priority for the settlement instruction execution.
     pub const SETTLEMENT_INSTRUCTION_EXECUTION_PRIORITY: Priority = 100;
+    /// Queue priority for multisig action execution.
+    pub const MULTISIG_PROPOSAL_EXECUTION_PRIORITY: Priority = 101;
+}
+
+/// Prefixes for scheduled actions
+pub mod schedule_name_prefix {
```

Comment:
If you define a generic name function then this `mod` could be private and the public interface might be an enum with all these variants.